<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User_managementClass</class>
 <widget class="QMainWindow" name="User_managementClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1261</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User_management</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1261</width>
      <height>731</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Downloads/User_management.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="Usernamelabel">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>195</y>
      <width>485</width>
      <height>46</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
    font-size: 14pt; /* Adjust the font size as needed */
    font-family: Georgia; /* Adjust the font family as needed */
}
</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="EmailLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>299</y>
      <width>485</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
    font-size: 14pt; /* Adjust the font size as needed */
    font-family: Georgia; /* Adjust the font family as needed */
}
</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>404</y>
      <width>485</width>
      <height>46</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
    font-size: 14pt; /* Adjust the font size as needed */
    font-family: Georgia; /* Adjust the font family as needed */
}
</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="PhoneNumberLabel">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>509</y>
      <width>485</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
    font-size: 14pt; /* Adjust the font size as needed */
    font-family: Georgia; /* Adjust the font family as needed */
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>590</y>
      <width>221</width>
      <height>68</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>691</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
    background-color: #FFFFFF; /* Set background color */
    alternate-background-color: #F6F5F3; /* Set alternate row background color */
    border: 1px solid #c0c0c0; /* Set border color */
    padding: 1px; /* Set cell padding */
}

QTableWidget::item {
    padding: 5px; /* Set padding for table items */
    font-size: 12px; /* Set font size */
    color: #333333; /* Set text color */
}

QHeaderView::section {
    background-color: #336A86; /* Set header background color */
    padding: 5px; /* Set header padding */
    font-size: 12px; /* Set header font size */
    color: #C8C8C8; /* Set header text color */
    border: 1px solid #c0c0c0; /* Set header border */
}

QTableWidget::item:selected {
    background-color: #a0a0a0; /* Set selected item background color */
    color: #ffffff; /* Set selected item text color */
}
</string>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>User Name</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Password</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>E-Mail</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Phone Number</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="RemoveButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>550</y>
      <width>221</width>
      <height>68</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Dashboardbutton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="manage_listingsButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1261</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
